File: pios.projekt.API/pios.projekt.models/Models/SchoolClass.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace pios.projekt.models.Models
{
    public class SchoolClass
    {
        public int Id { get; set; }
        public string Name { get; set; }

        public List<Student> students { get; set; } = new List<Student>();
        public List<TimetableRow> timetableRows { get; set; } = new List<TimetableRow>();

    }
}

File: pios.projekt.front/src/Models/SchoolClass.ts
import { Student } from './Student';
import { TimetableRow } from './TimetableRow';

export interface SchoolClass {
    id: number;
    name: string;
    students: Student[];
    timetableRows: TimetableRow[];
}

export class SchoolClassClass implements SchoolClass {
    id = 0;
    name = '';
    students = [];
    timetableRows = [];
}
